### ISSUE TITLE
Incorrect rls command in example settings

### ISSUE THREAD

**Issue by chetgurevitch** [autozimu/LanguageClient-neovim#41]
When installed through rustup the rls binary isn't actually added to ~/.cargo/bin/rls, I'm guessing there's one leftover on your system from building it locally and using cargo install which can be removed with 'cargo uninstall rls' .

The recommended way to run it now is 'rustup run nightly rls' b/c the binary is nested in one of rustup's toolchain folders which will differ by platform.

I'll submit a PR with a fix shortly.

**Comment by autozimu** [comment]
Merged in. Thanks.

**Comment by chetgurevitch** [comment]
No problem, also small correction apparantly the new installation method is supposed to create a proxy binary in ~/.cargo/bin (my bad), but this only works with the latest versions of rustup and is broken in at least Arch's official rustup package (despite it being up to date).

For now it looks like 'rustup run nightly rls' is still the best option as it works everywhere.

**Comment by autozimu** [comment]
Aha, that explains it.

When I made the change this morning, I was using Debain, on which I installed rustup using the official script.

Later I went to work, where I am using Arch (rustup from repo), and wondered why it doesn't work.

Thank you!

**Comment by chetgurevitch** [comment]
Happy to help! Also another update, I opened an issue on the arch bug tracker and it was fixed very quickly, everything's working with 1.2.0-4 although it looks like it might hit 1.2.0-6 by the time most mirrors have updated. 

It'd be a good idea to shorten the command to just 'rls' at some point given it should work with stable rust in the long run and the rest should no longer be necessary on any fully updated system. The only caveat is that the automated PATH modification during rustup installation is still broken, which affects more than just the rls and is something users should deal with as soon as possible if they haven't already.
